server/eb/app.py

```diff
@@ -105,7 +105,7 @@ def _before_adding_routes(app, app_config):
             "script-src": ["'self'", "'unsafe-eval'", "'unsafe-inline'"]
             + obsolete_browser_script_hash + script_hashes,
             "style-src": ["'self'", "'unsafe-inline'"],
-            "img-src": ["'self'", "data:", "https:///cellxgene.cziscience.com"],
+            "img-src": ["'self'", "data:", "'https:///cellxgene.cziscience.com'"],
```

Comment:
Ahh so tricky 🤦 Sorry for missing that too! Do we need single quotes for `data:`?

Reply:
Unit tests are still failing, so I guess, not sure why they weren't failing earlier ¯\_(ツ)_/¯

This also could be an issue of the source order.


```suggestion
            "img-src": ["'self'", "'data:'", "'https:///cellxgene.cziscience.com'"],
```

Reply:
Hmm yeah and `data:` doesn't seem to need quotes 🤔 

https://stackoverflow.com/questions/40360109/content-security-policy-img-src-self-data